src/test/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/CollectionExerciseServiceImplTest.java

```diff
@@ -402,8 +398,7 @@ public void testPatchCollectionExerciseUserDescription() throws Exception {
   public void testPatchCollectionExerciseNonUnique() throws Exception {
     CollectionExerciseDTO toUpdate = FixtureHelper.loadClassFixtures(CollectionExerciseDTO[].class).get(0);
     CollectionExercise existing = FixtureHelper.loadClassFixtures(CollectionExercise[].class).get(0);
-    Survey survey = FixtureHelper.loadClassFixtures(Survey[].class).get(0);
```

Comment:
Need to delete the test data class /src/test/resources/uk/gov/ons/ctp/response/collection/exercise/service/impl/CollectionExerciseServiceImplTest.Survey.json

Reply:
'Tis done